keps/sig-apps/4017-pod-index-label/kep.yaml

```diff
@@ -0,0 +1,37 @@
+title: Pod Index Label
+kep-number: 4017
+authors:
+  - "@danielvegamyhre"
+owning-sig: sig-apps
+status: implementable
+creation-date: 2023-05-16
+reviewers:
+  - "@ahg-g"
+  - "@smarterclayton"  
+approvers:
+  - "@wojtek-t"
+  - "@soltysh"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: beta
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.28"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha:
+  beta: "v1.28"
+  stable:
+
+feature-gates:
+  - name: PodIndexLabel
+    components:
+      - kube-apiserver
```

Comment:
Nit: this should be kube-controller-manager, since that's where you set the label, not apiserver.

Reply:
Fixed